src/Components/Book.js

```diff
@@ -10,23 +10,43 @@ const Book = (props) => {
   const dispatch = useDispatch();
 
   return (
-    <article>
-      <h2>{title}</h2>
-      <h3>{author}</h3>
-      <p>{category}</p>
-      <ul className={styles.book_btn_container}>
-        <li>
-          <button type="button">Comments</button>
-        </li>
-        <li>
-          <button type="button" onClick={() => dispatch(deleteBook(id))}>
-            Remove
-          </button>
-        </li>
-        <li>
-          <button type="button">Edit</button>
-        </li>
-      </ul>
+    <article className={styles.article}>
+      <div className={styles.book_details_main_container}>
+        <p>{category}</p>
+        <span>{title}</span>
+        <h3>{author}</h3>
+        <ul className={styles.book_btn_container}>
+          <li>
+            <button type="button">Comments</button>
+          </li>
+          <li>
+            <button type="button" onClick={() => dispatch(deleteBook(id))}>
+              Remove
+            </button>
+          </li>
+          <li>
+            <button type="button">Edit</button>
+          </li>
+        </ul>
```

Comment:
- [ ] Also, here we are missing the vertical line between the text as seen below
![Screenshot from 2023-05-18 14-02-50](https://github.com/bilalrajput09/Bookstore-react/assets/46388113/1dd5c0e1-a7ae-4e76-8184-7b00cd0c04b3)
![Screenshot from 2023-05-18 14-02-41](https://github.com/bilalrajput09/Bookstore-react/assets/46388113/152aa92d-a491-4ffa-a2f9-040e06c6545a)

Let us consider the text size and weight as well, let us kindly try to match the design